proposals/simd/SIMD.md

```diff
@@ -674,6 +679,14 @@ Load a `v128` vector from the given heap address.
 
 Load a single element and splat to all lanes of a `v128` vector.
 
+```python
+def S.load_splat(memarg):
+    result = S.New()
+    val = memory.bytes[memarg.start:memarg.start + S.LaneBytes])
+    for i in range(S.Lanes):
+        S.bytes[i:i + S.LaneBytes] = val
+    return result
```

Comment:
Why are we using `S.bytes[i:i + S.LaneBytes] = val`, and not `result[i]`?

Also, do we want to result S.splat?
```
def S.load_splat(memarg):
    val = memory.bytes[memarg.start:memarg.start + S.LaneBytes])
   return S.splat(val)
```